encoding/protobuf.go

```diff
@@ -39,8 +39,12 @@ func NewProtobuf(fullMethodName string, source protobuf.DescriptorProvider) (Ser
 
 	methodDescriptor := serviceDescriptor.FindMethodByName(methodName)
 	if methodDescriptor == nil {
-		//TODO: return a list of available methods
-		return nil, fmt.Errorf("service %q does not include a method named %q", serviceName, methodName)
+		available := make([]string, len(serviceDescriptor.GetMethods()))
```

Comment:
since the logic to build the message is adding noise to this function, i'd suggest adding a `findMethod` private method that will do the `serviceDescriptor.FindMethodByName` and build the error message as you do now

Reply:
Will change.